packages/libva.rb

```diff
@@ -1,26 +1,26 @@
 require 'package'
 
 class Libva < Package
-  description 'Libva is an implementation for VA-API (VIdeo Acceleration API)'
+  description 'Libva is an implementation for VA-API (Video Acceleration API)'
   homepage 'https://01.org/linuxmedia'
-  @_ver = '2.10.0'
-  version "#{@_ver}-2"
+  @_ver = '2.11.0'
+  version @_ver
   license 'MIT'
   compatibility 'all'
-  source_url "https://github.com/intel/libva/releases/download/#{@_ver}/libva-#{@_ver}.tar.bz2"
-  source_sha256 'fa81e35b50d9818fce5ec9eeeeff08a24a8864ceeb9a5c8e7ae4446eacfc0236'
+  source_url "https://github.com/intel/libva/archive/refs/tags/#{@_ver}.tar.gz"
```

Comment:
What's wrong with the previous release tag?

Reply:
They haven't generated a bz2 file on their release page.